### PR TITLE
added readme changes for collaborative stage view

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by Wajeed-msft — APPROVED** [review]

File: samples/tab-stage-view/csharp/README.md [hunk 1]
```diff
@@ -124,9 +124,9 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 ![Card Actions](TabInStageView/Images/cardActions.png)
 
-- Opening stage view from Adaptive Card Action.
+- Opening Collaborative Stage View. Please refer [Collaborative Stage view](https://review.learn.microsoft.com/en-us/microsoftteams/platform/tabs/tabs-link-unfurling?branch=pr-en-us-7891#collaborative-stage-view) for more details
 
-![Stage View](TabInStageView/Images/viaCardAction.png)
+![Tab View](TabInStageView/Images/CollaborativeStageView.png)
```
Comment (ht4963-ms) [inline samples/tab-stage-view/csharp/README.md:129]: It should say stage view, not "Tab view"

File: samples/tab-stage-view/csharp/README.md [hunk 2]
```diff
@@ -124,9 +124,9 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 ![Card Actions](TabInStageView/Images/cardActions.png)
 
-- Opening stage view from Adaptive Card Action.
+- Opening Collaborative Stage View. Please refer [Collaborative Stage view](https://review.learn.microsoft.com/en-us/microsoftteams/platform/tabs/tabs-link-unfurling?branch=pr-en-us-7891#collaborative-stage-view) for more details
```
Comment (ht4963-ms) [inline samples/tab-stage-view/csharp/README.md:127]: The link is a temporary link. Is that intended?

File: samples/tab-stage-view/csharp/README.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-stage-view/csharp/README.md]: changed

File: samples/tab-stage-view/csharp/README.md (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/tab-stage-view/csharp/README.md:127]: Yes, once the docs are published we will update the link

**Review by ht4963-ms — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build status
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5324966421

**Pull request merged**